Jenkinsfile_CNP

```diff
@@ -36,8 +41,8 @@ env.IDAM_URL = 'https://idam-api.aat.platform.hmcts.net'
 
 withPipeline(type, product, component) {
   enableAksStagingDeployment()
+  enableCrossBrowserTest(['chrome', 'firefox'])
```

Comment:
why are we enabling cross browser tests in all pipeline run?

Reply:
Its for test purposes removed now